core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1480,6 +1485,23 @@ public void run() {
         protected void afterAdd() throws IOException {}
     }
 
+    public void testTranslogCreateWithSmallSyncInterval() throws IOException, InterruptedException {
```

Comment:
I'm doubting whether we should have the test - i don't think it buys much . A reference to the build failure is enough?